docs/pages/access-graph/getting-started-on-prem.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Getting started with Teleport Access Graph (TAG)
+---
+
+<Notice type="note">
+Teleport Access Graph is provided by default to users of the Teleport Team plan.
+Visit the [signup page](https://goteleport.com/signup/) to begin your free&nbsp;trial.
+</Notice>
```

Comment:
Wanted to add this call-out, since this "Getting started" guide is the only one we have for now (there's no equivalent for Team/Cloud - but there's nothing to set up manually on Team either).

Reply:
This almost reads as if the first step to a "getting started on prem" guide is to sign up for a cloud trial.

Can we clarify by saying something like:

"Teleport Access Graph is enabled by default for Teleport Team users. You can skip the setup process here and start using Access Graph right away by signing up for a Teleport Team trial."